keyboards/silverbullet44/keymaps/FKML/keymap.c

```diff
@@ -0,0 +1,169 @@
+/* Copyright 2019 SwanMatch
```

Comment:
せっかくだから、ふかめるさんのCopyrightにしてみては?
```suggestion
/* Copyright 2020 FKML
```